go/base/decorators.py

```diff
@@ -0,0 +1,24 @@
+"""Decorator utilities."""
```

Comment:
Happy to just delete this. It's a nice idea but it's unrelated to this PR and currently unused.